components/collective-page/sections/Contribute.js

```diff
@@ -163,6 +164,8 @@ class SectionContribute extends React.PureComponent {
     const hasNoContributor = !this.hasContributors(contributors);
     const canContribute = collective.isActive && !isPastEvent(collective);
     const hasCustomContribution = !collective.settings?.disableCustomContributions;
+    const hasCryptoContribution =
+      collective.settings?.disableCryptoContributions === false && collective?.host?.settings?.cryptoEnabled;
```

Comment:
If collective has `settings={ }` then `hasCryptoContribution` will be false (because `collective.settings?.disableCryptoContribution` => `undefined`). If we want to make this opt-out, it should look like:
```suggestion
      !collective.settings?.disableCryptoContributions && collective.host?.settings?.cryptoEnabled;
```

Reply:
At this stage, we want to have the following. Consistent with what we have already somewhere else:

get(collective ,'settings.disableCryptoContributions', true);